stdeb.cfg

```diff
@@ -1,5 +1,5 @@
 [colcon-package-selection]
 No-Python2:
-Depends3: python3-colcon-core
+Depends3: python3-colcon-core (>= 0.3.19)
```

Comment:
I just noticed that you hadn't done this for `colcon-core`. Should I revert this part?

Reply:
> I just noticed that you hadn't done this for `colcon-core`.

Sorry, I don't get what you are referring to here. Can you please explain in more detail.

Reply:
Never mind, I was mistaken, my apologies. All good here.